docs/mergeduty/howto.md

```diff
@@ -124,12 +133,12 @@ when: this can happen before relman request migration. e.g. the same day but in
 1. The reconfiguration is triggered by a cron job every hour exactly on the hour (`0 * * * *` in crontab). Your options are:
    * Wait for the reconfig to happen via cron.
    * Ask buildduty to manually trigger it.
-   * Run it yourself, see [below](#appendix-perform-a-manual-reconfig) for instructions
+   * Run it yourself, see [how-to-manually-perform-reconfig](https://github.com/mozilla-releng/releasewarrior-2.0/blob/master/docs/misc-operations/manually-perform-bb-reconfig-generate-partials.md) for instructions
```

Comment:
Do we still need this, and the other buildbot-config sections of this doc?

Reply:
Sounds like bbconfigs gecko_version is useless now, I agree. But we still need it until esr52 dies, dont't we? Actually, to be fully honest, I never understood gecko_versions.json in bbconfigs' role.

Reply:
I think it controls some builder additiona/removal. In any case, now that we won't be doing any more merges involving Buildbot - that's not relevant for merge day.